tests/functional/pair.phpt

```diff
@@ -19,7 +19,7 @@ var_dump($first);
 var_dump($second);
 
 --EXPECTF--
-(5, in manus tuas domine commendo spiritum meum)
-(repetitio est mater studiorum, 10)
+(5, "in manus tuas domine commendo spiritum meum")
```

Comment:
This output should not change.
That is the same output I'd have in kotlin.

Reply:
I am moving this change to #8